### DISCUSSION TITLE
"player is not whitelisted" is not working?

### DISCUSSION THREAD

**Discussion by udukiukia** [SkriptLang/Skript#6299] [General]
```v
command /test:
	trigger:
		if player is whitelisted:
			send "member"
		else if player is not whitelisted:
			send "not member"
		else:
			send "error"
```

<img width="279" alt="スクリーンショット 2024-01-05 23 46 36" src="https://github.com/SkriptLang/Skript/assets/131680691/d43b4581-792c-4cc1-83db-f2cc4f92c12a">

Why does "member" appear when the player is on the whitelist, but "error" appears when the player is not on the whitelist?

MC 1.20.2
Skript v2.7.3

**Comment by NotSoDelayed** [comment]
Fixed in #5422 and awaiting for further review.

**Comment by AyhamAl-Ali** [comment]
Created an issue for it #6300
